index.md

```diff
@@ -26,25 +26,30 @@ We want LRUG to be a diverse and inclusive community, united by an interest in R
      * [Timing](#timing)
    * [After The Talks](#after-the-talks)
  * [Sponsorship](#sponsorship)
+   * [Meeting Sponsorship](#meeting-sponsorship)
+   * [Other Sponsorship Opportunities](#other-sponsorship-opportunities)
+     * [Sponsor a give-away](#sponsor-a-give-away)
+     * [Sponsor food (pizza)](#sponsor-food-pizza)
+     * [Sponsor drinks](#sponsor-drinks)
  * [About This Document](#about-this-document)
    * [License](#license)
 
 ## Code of Conduct
 
-It’s of primary importance that everyone who wants to participate feels safe and welcome. When participating in LRUG, either online or offline, at Skills Matter or at the pub afterwards, we expect you to:
+It’s of primary importance that everyone who wants to participate feels safe and welcome. When participating in LRUG, either online or offline, at an in-person event, or at the pub afterwards, we expect you to:
```

Comment:
[nitpick]

Since we say `either online or offline`, can we do without `at an in-person event`? At least for me, it is redundant when we already talk about `offline`.

Reply:
I think that's fair.  I might tweak the wording a bit more - we use "offline" and "online" elsewhere so there's some good reinforcement about using those terms, but something feels more explicit about "in-person".  I'll have a think.

Reply:
I _think_ what the original was getting at is that "offline" doesn't just mean the meetings, but also the pub afterwards, and I think that's important to preserve.  I tried:

> When participating in LRUG, either online (e.g. via [the mailing list](https://lrug.org/mailing-list), at a [remote meeting](#meetings), or via any other online platform) or offline (e.g. at an [in-person event](#meetings), or [at the pub afterwards](#after-the-talks)), we expect you to:

While that tries to capture more explicitly what we mean by "online" and "offline" I don't love the parenthesis and "e.g.".  Feels like it's too cumbersome to enumerate things.  Maybe:

> When participating in LRUG, either online or in-person at [a meeting](#meetings) or [the pub afterwards](#after-the-talks), we expect you to:

It's less explicit on the online side, but maybe that's ok.  WDYR?

Reply:
My first instinct was to simply say `When participating in LRUG, either online or offline, we expect you to:`

But, if you want to be more explicit, what about ` When participating in LRUG, online, at an in-person meeting, or at the pub after a meeting, we expect you to:`?

Reply:
:+1: that's basically what I've gone with - thanks for the thoughts!